homeassistant/components/yamaha_musiccast/translations/select.en.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Please rename this file to `homeassistant/components/yamaha_musiccast/strings.select.json`